tools/yaml-templates/ios-test.yml

```diff
@@ -147,11 +147,16 @@ steps:
 
       echo "ls -lR ${output}"
       ls -lR "${output}"
+
+      echo "Zipping logs directory..."
+      cd "$(agent.buildDirectory)/Logs/$(agent.JobName)"
```

Comment:
>$(agent.buildDirectory)/Logs/$(agent.JobName) [](http://example.com/codeflow?start=10&length=45)

It looks like there is a variable already set to this above (`output`), can that variable be reused here?

Reply:
Yes, it will be reused.